src/shared/theme.tsx

```diff
@@ -1,12 +1,14 @@
 type theme = 'light' | 'dark';
 
 const timeToTheme = (): theme => {
-  const time = new Date().getHours();
-  if (time > 18 || time < 7) {
-    return 'dark';
-  } else {
-    return 'light';
-  }
+  // const time = new Date().getHours();
+  // if (time > 18 || time < 7) {
+  //   return 'dark';
+  // } else {
+  //   return 'light';
+  // }
+
+  return 'dark';
```

Comment:
Is the login form designed only for the dark theme?

Reply:
Yeah... I didn't want to import this theme variable and have if statements everywhere, so I rendered it useless for the time being

Reply:
I don't know why we didn't just use a media query to begin with, but I guess that'll be for a separate PR.